src/util/addBoxPlot.js

```diff
@@ -1,30 +1,20 @@
-export default function addBoxplot
-    (svg
-    ,results
-    ,height
-    ,width
-    ,domain
-    ,boxPlotWidth
-    ,boxColor
-    ,boxInsideColor
-    ,format = '.2f'
-    ,horizontal = true) {
+export default function addBoxplot(chart, group
```

Comment:
honestly, if we're going to take it this far. it might be better just to send it the chart object and nothing else. Just have `group`, `boxInsideColor` and `precision` as options for the user there.